src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -107,6 +107,7 @@
     <Compile Include="$(CommonTestPath)\System\PerfUtils.cs">
       <Link>Common\System\PerfUtils.cs</Link>
     </Compile>
+    <Compile Condition="'$(TargetGroup)'=='netstandard1.7'" Include="netstandard.1.7.Tests.cs" />
```

Comment:
Agreed, we will have to fix this eventually.

---
In reply to: [72679281](https://github.com/dotnet/corefx/pull/10297#discussion_r72679281) [](ancestors = 72679281)